singer_sdk/sql/__init__.py

```diff
@@ -0,0 +1,5 @@
+"""Module for SQL-related operations."""
+
+from .connector import SQLConnector
```

Comment:
@kgpayne - It feels just a little odd to me that `SQLStream` and `SQLSink` are not in the `sql` module. 

What do you think of reversing the module ref name, from `singer_sdk/sql/connector.py` to `singer_sdk/connectors/sql.py` - which then would make `SQLConnector` importable from `singer_sdk.connectors` and also `singer_sdk.connectors.sql`. Not a ton of other uses for this namespace as of now, but this leaves open the possibility us us creating something like `singer_sdk.connectors.OpenAPIConnector` and similar.

Alternatively, we could put this under `singer_sdk/helpers/sql.py`.

What do you think?

cc @edgarrmondragon, @cjohnhanson

Reply:
I really like `singer_sdk/connectors/sql.py`. It's similar to what I imagined for https://github.com/meltano/sdk/issues/317 😄